src/pyskindose/analyze_data.py

```diff
@@ -56,9 +56,11 @@ def analyze_data(
     if not plot_dose_map:
         return output
 
-    create_dose_map_plot(
-        patient=patient, settings=settings,
-        dose_map=output[c.OUTPUT_KEY_DOSE_MAP]
-    )
+    if settings.mode == c.MODE_CALCULATE_DOSE:
```

Comment:
What should happen here if `settings.mode != c.MODE_CALCULATE_DOSE`?

If this is the only setting allowed here you should raise an exception if `settings.mode` has another value

Reply:
Resolved in fa2ee270b67e37a926235a83d687cf7fc0897188